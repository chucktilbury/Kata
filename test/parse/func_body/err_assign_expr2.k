;-------------------------------------------------------------------------------
; Assignment expression cannot have assignemnt
;-------------------------------------------------------------------------------

var string name1 = (asdf = 1) * 10
