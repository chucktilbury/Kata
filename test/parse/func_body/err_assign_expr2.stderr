Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_assign_expr2.k: 5: 27: expected a ')' but got =
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_assign_expr2.k: 5: 35: expected an assignment item but got literal signed
